build.sbt

```diff
@@ -231,14 +227,25 @@ lazy val sparkNotebookCore = Project(id = "spark-notebook-core", base = file("mo
     Extra.sparkNotebookCoreSettings
   )
 
+lazy val gitNotebookProvider = Project(id = "git-notebook-provider", base = file("modules/git-notebook-provider"))
+  .settings(
+    version := version.value
+  )
+  .settings(sharedSettings: _*)
+  .dependsOn(sparkNotebookCore)
+  .settings(
+    Extra.gitNotebookProviderSettings
+  )
+
 lazy val sparkNotebook = project.in(file(".")).enablePlugins(play.PlayScala).enablePlugins(SbtWeb)
-  .aggregate(sparkNotebookCore, sbtDependencyManager, subprocess, observable, common, spark, kernel)
-  .dependsOn(sparkNotebookCore, subprocess, observable, common, spark, kernel)
+  .aggregate(sparkNotebookCore, gitNotebookProvider, sbtDependencyManager, subprocess, observable, common, spark, kernel)
+  .dependsOn(sparkNotebookCore, gitNotebookProvider, subprocess, observable, common, spark, kernel)
   .settings(sharedSettings: _*)
   .settings(
-    bashScriptExtraDefines <+= (version, scalaBinaryVersion, scalaVersion, sparkVersion, hadoopVersion, withHive) map { (v, sbv, sv, pv, hv, wh) =>
-      """export ADD_JARS="${ADD_JARS},${lib_dir}/$(ls ${lib_dir} | grep common.common | head)""""
+    bashScriptExtraDefines <+= (organization, version, scalaBinaryVersion, scalaVersion, sparkVersion, hadoopVersion, withHive) map { (org, v, sbv, sv, pv, hv, wh) =>
+      s"""export ADD_JARS="$${ADD_JARS},$${lib_dir}/$$(ls $${lib_dir} | ${org}.common | head)""""
```

Comment:
I guess this expression: `ls $${lib_dir} | ${org}.common | head` will give some "command not found" error 
I think the `grep` is missing here when compared to the old version.